SparkPost/2web2ui
# AC-1969: Global tracking for all forms in abtest and account

AC-1969: Global tracking for all forms in abtest and account

### What Changed

- Send the Form data for abtest and account to Segment.
- Create a allowlist for form ids whose data we are sending to segment
- Added helpers for converting the form submission events to JSON data that can be sent to segment.
- Added unit tests for these helpers.
- Added cypress tests for abtest form 
- Checked all form in abtesta and account and made sure the form submit send the correctly parsed data to segment

### How To Test

- Checkout forms on abtest and see if the segment is receiving correctly parsed data
   1. /ab-testing/:name/:version (EditPage)
   2. /ab-testing/create (CreatePage)
- Checkout forms on /account/settings and see if the segment is receiving correctly parsed data 
   1. Click on Provision SSO and check if form submits correctly
- Check if data of all other forms that are not in allowlist is not sent to segment

### To Do
